build/build_openr.sh

```diff
@@ -7,11 +7,17 @@
 # LICENSE file in the root directory of this source tree.
 #
 
+GETDEPS="$(dirname "$0")/fbcode_builder/getdeps.py"
+INSTALL_PREFIX="/opt/facebook"
 PYTHON3=$(command -v python3)
 
 if [ "$PYTHON3" == "" ]; then
   echo "ERROR: No \`python3\` in PATH"
   exit 1
 fi
 
-python3 build/fbcode_builder/getdeps.py build openr
+# TODO: Get apt deb installing working - dies in Docker container with libzstd-dev fixed
+# python3 "$GETDEPS" --allow-system-packages install-system-deps --recursive openr
+python3 "$GETDEPS" --allow-system-packages build --no-tests --install-prefix "$INSTALL_PREFIX" openr
+# TODO: Maybe fix src-dir to be absolute reference to dirname $0's parent
+python3 "$GETDEPS" fixup-dyn-deps --strip --src-dir=. openr _artifacts/linux  --project-install-prefix openr:"$INSTALL_PREFIX" --final-install-prefix "$INSTALL_PREFIX"
```

Comment:
This should populate `_artifacts/linux` with  the bits that you would install to `/opt/facebook`.

For `docker_openr_help_test.sh` to run, I think you may "just" need to run `build_openr.sh` first, then:

```
LD_LIBRARY_PATH=_artifacts/linux/lib PATH=_artifacts/linux/sbin:$PATH openr_bin --help
```